.gitlab/pipeline/check.yml

```diff
@@ -128,19 +131,61 @@ check-runtime-migration-westend:
     - .check-runtime-migration
   variables:
     NETWORK: "westend"
+    PACKAGE: "westend-runtime"
+    WASM: "westend_runtime.wasm"
+    URI: "wss://westend-try-runtime-node.parity-chains.parity.io:443"
+    EXTRA_ARGS: "--no-weight-warnings"
 
 check-runtime-migration-rococo:
   stage: check
-  # DAG
-  needs:
-    - job: check-runtime-migration-westend
-      artifacts: false
   extends:
     - .docker-env
     - .test-pr-refs
     - .check-runtime-migration
   variables:
     NETWORK: "rococo"
+    PACKAGE: "rococo-runtime"
+    WASM: "rococo_runtime.wasm"
+    URI: "wss://rococo-try-runtime-node.parity-chains.parity.io:443"
+    EXTRA_ARGS: "--no-weight-warnings"
+
+# Check runtime migrations for Parity managed asset hub chains
+check-runtime-migration-asset-hub-westend:
+  stage: check
+  extends:
+    - .docker-env
+    - .test-pr-refs
+    - .check-runtime-migration
+  variables:
+    NETWORK: "asset-hub-westend"
+    PACKAGE: "asset-hub-westend-runtime"
+    WASM: "asset_hub_westend_runtime.wasm"
+    URI: "wss://westend-asset-hub-rpc.polkadot.io:443"
```

Comment:
Where does this `WASM` name get set? Usually we have something with the version like `westmint_runtime-v9430.compact.compressed.wasm`

Reply:
I'm guessing in `build.rs`? 

When I compile a runtime, 3 WASM files are produced under `target/release/wbuild/{runtime-crate-name}`:
- `{runtime_name}.wasm`
- `{runtime_name}.compact.wasm`
- `{runtime_name}.compact.compressed.wasm`

I've never seen a version in any .wasms. Perhaps it's added manually by the RelEng team?

Reply:
OK yeah that must be part of the releng process. But why are you using the `.wasm`? We use `{runtime_name}.compact.compressed.wasm` on-chain so I think it makes more sense to use that.

Reply:
I used the `.wasm` because I didn't think there would be any differences in behavior for the purposes of dry-running runtime upgrades. I'll change it though just in case there's some edge case.